labelbox/schema/project.py

```diff
@@ -216,17 +217,35 @@ def set_labeling_parameter_overrides(self, data):
 
             >>> project.set_labeling_parameter_overrides([
             >>>     (data_row_1, 2, 3), (data_row_2, 1, 4)])
+            OR
+            >>> project.set_labeling_parameter_overrides([
+            >>>     (data_row_1.uid, 2, 3), (data_row_2.uid, 1, 4)])
 
         Args:
             data (iterable): An iterable of tuples. Each tuple must contain
-                (DataRow, priority, numberOfLabels) for the new override.
+                (DataRow or DataRow.uid, priority, numberOfLabels) for the new override.
```

Comment:
Nitpick: the codebase generally tries to adhere to max 80 characters per line, and Google Style guide in general:
http://google.github.io/styleguide/pyguide.html#32-line-length